client/src/components/CardDialog/CardDialog.tsx

```diff
@@ -11,34 +11,31 @@ import {
   DialogContent,
   TextField,
 } from '@material-ui/core';
-import cardDialogStyles from './cardDialogStyles';
-import useColorTagStyles from '../Kanban/shared/colorStyles';
-import { useKanban } from '../../context/useKanbanContext';
 import ClearIcon from '@material-ui/icons/Clear';
-import AssignmentOutlinedIcon from '@material-ui/icons/AssignmentOutlined';
 import ScheduleIcon from '@material-ui/icons/Schedule';
 import ChatBubbleOutlineIcon from '@material-ui/icons/ChatBubbleOutline';
+import ImportContactsOutlinedIcon from '@material-ui/icons/ImportContactsOutlined';
 import DatePicker from '../DatePicker/DatePicker';
 import { IColumn } from '../../interface/Column';
-import { useSnackBar } from '../../context/useSnackbarContext';
 import DialogItemGroup from './DialogItemGroup/DialogItemGroup';
-import { useDialog } from '../../context/useDetailContext';
-import ImportContactsOutlinedIcon from '@material-ui/icons/ImportContactsOutlined';
+import cardDialogStyles from './cardDialogStyles';
+import useColorTagStyles from '../Kanban/shared/colorStyles';
+import { useSnackBar, useDialog, useKanban } from '../../context/';
 
-type DialogProps = {
+interface DialogProps {
   name: string;
   tag: string;
   columnId: string;
   id: string;
-};
+}
 
 const CardDialog = ({ name = 'blank', columnId, tag = 'white', id }: DialogProps): JSX.Element => {
```

Comment:
@street-jackal Just using the interface (`DialogProps` )here instead of declaring a type to hold the interface